modules/op-running-pipelines-in-restricted-environment.adoc

```diff
@@ -0,0 +1,200 @@
+// Module included in the following assemblies:
+//
+// pipelines/creating-applications-with-cicd-pipelines
+
+[id="op-running-pipelines-in-restricted-environment_{context}"]
+=  Running pipelines in a restricted environment
+
+
+To run OpenShift Pipelines in a disconnected cluster or a cluster provisioned in a restricted environment, you must ensure that either the Samples Operator is configured for a restricted network, or a cluster administrator has created a cluster with a mirrored registry. To learn more about Samples Operator and mirrored registries, see the Additional resources section.
+
+The following section uses the `Pipelines Tutorial` example to create a pipeline for an application in a restricted environment using a cluster with a mirrored registry. To ensure that the `pipelines-tutorial` example works in a restricted environment, you must mirror the respective builder images from the mirror registry for the front-end interface, `vote-ui`, back-end interface, `vote-api`, and the `cli`.
+
+.Procedure
+
+
+. Mirror the builder image from the mirror registry for the front-end interface, `vote-ui`:
```

Comment:
Can change this from `:` to `.` when it's not leading in to an actual codeblock/command (since this is going into a substep list).